### PR TITLE
feat(default-theme): added custom form styles file

### LINKED ISSUE #984: [FEATURE] Create common styles for Forms
### Acceptance criteria

- [ ] create `assets/scss/forms.scss` file with form styling
- [ ] move existing form styles there and make them common
- [ ] test if user can override this in his project (@patzick will help here)

### PR DESCRIPTION
**Pull request by Malwurf**
## Changes
Added new common file with default forms styles
Moved styles from other components with forms to the new one
 closes #984 

### Checklist

- [x] I followed [contributing](https://github.com/DivanteLtd/shopware-pwa/blob/master/CONTRIBUTING.md) guidelines

### CODE REVIEW

File: packages/default-theme/components/checkout/steps/PaymentStep.vue [hunk 1]
```diff
@@ -116,51 +116,51 @@ export default {
...
-      &--half {
-        flex: 1 1 50%;
-        &-even {
-          padding: 0 0 0 var(--spacer-lg);
-        }
-      }
-    }
-  }
-}
+// .form {
```
Comment (patzick) [inline packages/default-theme/components/checkout/steps/PaymentStep.vue:119]: to delete

File: packages/default-theme/components/checkout/steps/ShippingStep.vue [hunk 2]
```diff
@@ -131,48 +131,48 @@ export default {
...
-    }
-    &__action-button {
-      --button-font-weight: var(--font-normal);
-      &:first-child {
-        margin: 0 var(--spacer-lg) 0 0;
-      }
-    }
-  }
-}
+// .form {
```
Comment (patzick) [inline packages/default-theme/components/checkout/steps/ShippingStep.vue:134]: same

File: packages/default-theme/components/checkout/steps/guest/BillingAddressGuestForm.vue [hunk 3]
```diff
@@ -188,52 +183,52 @@ export default {
...
-      &--half {
-        flex: 1 1 50%;
-        &-even {
-          padding: 0 0 0 var(--spacer-lg);
-        }
-      }
-    }
-  }
-}
+// .form {
```
Comment (patzick) [inline packages/default-theme/components/checkout/steps/guest/BillingAddressGuestForm.vue:186]: same

File: packages/default-theme/assets/scss/forms.scss [hunk 4]
```diff
@@ -0,0 +1,101 @@
+.form {
```
[the remark this answers is not in the document]
Comment (Malwurf) [reply packages/default-theme/assets/scss/forms.scss]: Changed in all occurrences

File: packages/default-theme/assets/scss/main.scss [hunk 5]
```diff
@@ -1,5 +1,6 @@
 @import "~@storefront-ui/vue/styles";
 @import "@/assets/scss/variables.scss";
+@import "forms.scss";
```
[the remark this answers is not in the document]
Comment (Malwurf) [reply packages/default-theme/assets/scss/main.scss]: Removed from main.scss and imported to all component with forms.

File: packages/default-theme/components/checkout/steps/PaymentStep.vue (hunk 1 above)
Comment (Malwurf) [reply packages/default-theme/components/checkout/steps/PaymentStep.vue]: Removed unnecessary comment

File: packages/default-theme/components/checkout/steps/ShippingStep.vue (hunk 2 above)
Comment (Malwurf) [reply packages/default-theme/components/checkout/steps/ShippingStep.vue]: Removed unnecessary comment

File: packages/default-theme/components/checkout/steps/guest/BillingAddressGuestForm.vue (hunk 3 above)
Comment (Malwurf) [reply packages/default-theme/components/checkout/steps/guest/BillingAddressGuestForm.vue]: Removed unnecessary comment

**Pull request merged**